tests/unit/test_merge_spdx.py

```diff
@@ -0,0 +1,517 @@
+from itertools import chain
+from typing import cast
+
+import pytest
+
+from cachi2.core.models.sbom import (
+    SPDXPackage,
+    SPDXPackageExternalRefPackageManagerPURL,
+    SPDXRelation,
+    SPDXSbom,
+)
+from cachi2.interface.cli import merge_relationships
+
+
+@pytest.mark.parametrize(
```

Comment:
what would the ids be like? Merging spdx1-to-spdx2, merging spdx2-to-spdx1? In that case I don't think it's necessary, if you look at it for 3 seconds it should be clear to you. I can add doc string to the test if you need it and describe it there. But I don't think any ids which shows in the test could help you quickly understand the difference. 
Both asserts test correctness of the output therefore they should be in the same test. Packages are sorted in order to more easily test packages are in match with expected packages. As when you switch the order of two documents it can also change order of two packages.